prow/config.yaml

```diff
@@ -516,6 +516,35 @@ presubmits:
       - name: docker-graph
         hostPath:
           path: /mnt/disks/ssd0/docker-graph
+  kubernetes/kube-deploy:
+  - name: pull-kube-deploy-build
+    agent: kubernetes
+    always_run: true
+    context: pull-kube-deploy-build
+    rerun_command: "/test pull-kube-deploy-build"
+    trigger: "/test( all| pull-kube-deploy-build)"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
```

Comment:
I just picked an image that seemed like it might work. I don't know how to test this. The only thing these jobs depend on is go 1.9+

Reply:
ref https://github.com/kubernetes/test-infra/blob/master/images/gcloud/Dockerfile, yeah it's go 1.9.3 with an ancient gcloud